src/components/layout/MenuBar.tsx

```diff
@@ -1,54 +1,45 @@
-import home from '../../components/icons/home_738822.png';
+// import home from '../../components/icons/home_738822.png';
 import logo from '../../turing-logo-gray.png';
-import person from '../../components/icons/person-icon.png';
-import building from '../../components/icons/building-1062.png';
-import plus from '../../components/icons/plus-small_4338829.png';
-import papers from '../../components/icons/documents-symbol_35920.png';
-
+// import person from '../../components/icons/person-icon.png';
+import PersonIcon from '@mui/icons-material/Person';
+import PersonOutlineOutlinedIcon from '@mui/icons-material/PersonOutlineOutlined';
+// import building from '../../components/icons/building-1062.png';
+import ApartmentIcon from '@mui/icons-material/Apartment';
+// import plus from '../../components/icons/plus-small_4338829.png';
+import PersonAddAlt1Icon from '@mui/icons-material/PersonAddAlt1';
+// import papers from '../../components/icons/documents-symbol_35920.png';
+import DescriptionIcon from '@mui/icons-material/Description';
+import HomeIcon from '@mui/icons-material/Home';
```

Comment:
It looks like you went ahead and installed MUI on this branch, is that correct?

Reply:
That is correct, yes! You discussed it in slack with me, and I followed your advice.

Reply:
I thought from our conversations that I was going to tackle this, but thank you for completing it.